crates/blockifier/src/execution/syscalls/mod.rs

```diff
@@ -55,8 +55,7 @@ pub mod hint_processor;
 mod secp;
 
 #[cfg(test)]
-#[path = "syscalls_test.rs"]
-pub mod syscalls_test;
+pub mod syscall_tests;
```

Comment:
I know we have it different in our blockifier but I think we should keep the same name for the module "syscalls_test"